siege_utilities/reporting/idml_export.py

```
@@ -417,9 +417,22 @@ def _build_spread(self, spread_id: str, story_ids: List[str]) -> str:
                 f'ParentStory="{sid}" '
                 f'GeometricBounds="{bounds}"/>'
             )
+        from pathlib import Path
+        from urllib.parse import quote
+        from xml.sax.saxutils import quoteattr
+
         for img in self._image_frames:
             sid = story_ids[idx]; idx += 1
```

Ruff (0.15.12)

[error] Multiple statements on one line (semicolon)

(E702)